src/test/js/multibranch/testResults.js

```diff
@@ -27,9 +27,14 @@ module.exports = {
     },
 
     'Open acitivty page wait for first run to finish': (browser) => {
+        
+
         const blueActivityPage = browser.page.bluePipelineActivity().forJob(jobName);
+        blueActivityPage.click(".branches");
+        blueActivityPage.click(".activity");
```

Comment:
Yes, it probably should as elsewhere I have seen it fail on clicking things implying that it doesn't wait for it to be visible.